### ISSUE TITLE
Adagrad learning rate

### ISSUE THREAD

**Issue by berryweinst** [facebookresearch/dlrm#82]
Can you please note what is the learning rate used to train small Criteo with Adagrad in order to achieve the paper's convergence pattern?

**Comment by mnaumovfb** [comment]
As far as I recall the Adagrad learning rate was 0.01.
